static/js/components/VideoPlayer_test.js

```diff
@@ -225,17 +235,6 @@ describe("VideoPlayer", () => {
     assert.isDefined(wrapper.find(".vjs-playback-rate-value"))
   })
 
-  it("toggleFullScreen causes player to dispatchEvent", () => {
-    const wrapper = renderPlayer()
-    wrapper.instance().player = playerStub
-    // $FlowFixMe
-    containerStub.parentElement[FULLSCREEN_API.requestFullscreen] = () => {}
-    wrapper.instance().videoContainer = containerStub
-
-    wrapper.instance().toggleFullscreen()
-    sinon.assert.calledWith(wrapper.instance().player.el_.dispatchEvent)
-  })
-
   it("toggleFullScreen on causes player to dispatchEvent", () => {
```

Comment:
was this just accidentally duplicated?

Reply:
Yes, it was a dupe, not testing anything not already covered.